third_party/move/move-vm/types/src/loaded_data/runtime_types.rs

```diff
@@ -409,6 +409,30 @@ impl Type {
         Ok(())
     }
 
+    /// Returns true if the type is a signer or an immutable signer reference type. Returns false
+    /// otherwise.
+    pub fn is_signer_or_signer_ref_ty(&self) -> bool {
```

Comment:
nit: because this is a method on `Type`, we can omit the suffix `_ty` in the method name.